### DISCUSSION TITLE
Number of finders filter

### DISCUSSION THREAD

**Discussion by aviggiano** [solodit/feedback#12] [New Feature Requests]
The filters from "number of finders" are not technically correct, since "Less than" will show results with a number up to the value provided and "More than" will show results with a number at least the value provided.

For example, less than 2 finders will return reports with 1 or 2 finders, and not 1 as expected.

**Comment by wang1252** [comment]
@aviggiano 
We will fix as your feedback.
Thank you.

**Comment by hans-cyfrin** [comment]
Closing as fixed.
